Gemfile.lock

```diff
@@ -1,45 +0,0 @@
-GEM
```

Comment:
I'd say let's keep it - in line with reqs from people who know more than me. If someone give like actionable failure dets it can always die later :)

Thanks for updates will check it out tomorrow